src/formulas/parser.ts

```diff
@@ -171,7 +171,7 @@ function parseFunctionArgs(tokens: Token[]): AST[] {
     consumeOrThrow(tokens, "COMMA", _lt("Wrong function call"));
     args.push(parseOneFunctionArg(tokens));
   }
-  consumeOrThrow(tokens, "RIGHT_PAREN", _lt("Wrong function call"));
+  consumeOrThrow(tokens, "RIGHT_PAREN", _lt("Missing closing parenthesis"));
```

Comment:
I think it's worth to add a test for this case and the case L144, as it does not seem to be covered

Reply:
It's actually impossible to have this error message raised from here.. Because of the while condition, the next token *is* a right parenthesis. I removed the message since it's useless.
I added a test for L144